snow/consensus/snowman/traced_consensus.go

```diff
@@ -29,16 +29,6 @@ func Trace(consensus Consensus, tracer trace.Tracer) Consensus {
 	}
 }
 
-func (c *tracedConsensus) Add(ctx context.Context, blk Block) error {
-	ctx, span := c.tracer.Start(ctx, "tracedConsensus.Add", oteltrace.WithAttributes(
-		attribute.Stringer("blkID", blk.ID()),
-		attribute.Int64("height", int64(blk.Height())),
-	))
-	defer span.End()
-
-	return c.Consensus.Add(ctx, blk)
-}
```

Comment:
Why are we removing the context and tracing from `Add` here?

Reply:
The only reason `ctx` was added to this function was to pass it through into `Reject`. When we added tracing we only added tracing for methods that had `ctx` anyway... We could re-introduce it and just drop the context from the implementation of `Add` if we think the tracing is important here.